app/firebaseConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore"
+
+const firebaseConfig = {
+    apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
```

Comment:
can this be made public with `NEXT_PUBLIC` ?

Reply:
No